Kitodo/src/main/java/org/kitodo/production/helper/cache/RequestScopeCacheHelper.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.helper.cache;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Supplier;
+
+import javax.faces.context.FacesContext;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Caches arbitrary objects while processing a request.
+ * 
+ * <p>Stores objects in a simple hash map inside FacesContext.getAttributes(). After each request, 
+ * a new FacesContext will be created, which means cached objects are not available any more.</p>
+ * 
+ * <p>Usually, such caching is not necessary when implementing Beans annotated with @RequestScope
+ * and getters that store processed values in local variables. However, a lot of beans do not 
+ * follow this pattern, which means that some common methods, e.g., getting the current locale of 
+ * a user (which requires a database lookup), are re-evaluated many times during processing of 
+ * a request.</p>
+ * 
+ * <p>Therefore, this class provides the possibility to inject a request scoped cache.</p>
+ * 
+ * <p>If this cache is used outside of a JSF request context, it will simply re-evaluate the supplier 
+ * function upon every request.</p>
+ * 
+ */
+public class RequestScopeCacheHelper {
+
+    private static final String ATTRIBUTE_NAME = "RequestScopeCacheHelper";
+
+    @SuppressWarnings("unused")
+    private static final Logger logger = LogManager.getLogger(RequestScopeCacheHelper.class);
```

Comment:
Do we need this logger? If not then I suggest to remove it.

Reply:
It's not needed. Removed